### PR TITLE
Invoke Script

### PR DESCRIPTION
**Pull request by sunbryely-influxdata**
Invoke Script endpoint

### CODE REVIEW

File: src/svc/invocable-scripts/paths/scripts_scriptID_invoke.yml [hunk 1]
```diff
@@ -18,11 +44,85 @@ post:
...
+          schema:
+            $ref: "../../../common/schemas/Error.yml"
+          examples:
+            invalidParameters:
+              summary: Invalid parameters have been passed into the script.
+              value: {
+                "code": "invalid",
+                "message": "invalid parameters provided"
+              }
+    "401":
```
Comment (sunbryely-influxdata) [inline src/svc/invocable-scripts/paths/scripts_scriptID_invoke.yml:77]: @jstirnaman 
When I `run make generate-all` i get this reference error
`Error opening file "/openapi/src/svc/invocable-scripts/schemas/AuthorizationError.yml" 
ENOENT: no such file or directory, open '/openapi/src/svc/invocable-scripts/schemas/Error.yml'
`

I added the file into the svc schemas folder to resolve the issue but I dont think thats a good solution. Additionally, I get a ref error in the docs-v2 when running the bundling command.

`[4] cloud/ref.yml:11146:11 at #/paths/~1api~1v2~1scripts~1{scriptID}~1invoke/post/responses/401

Can't resolve $ref

11144 |         type: string
11145 | "401":
11146 |   $ref: '#/paths/~1scripts/get/responses/401'
11147 | "404":
11148 |   content:
`

Would you happen to know whats causing this or what I might be doing incorrectly?

File: src/svc/invocable-scripts/schemas/ScriptInvocationParams.yml [hunk 2]
```diff
@@ -3,3 +3,7 @@ properties:
   params:
     type: object
     additionalProperties: true
+    description: |
+      Script parameters.
+      Provide key-value pairs within this params object for parameters
```
Comment (jstirnaman) [inline src/svc/invocable-scripts/schemas/ScriptInvocationParams.yml:8]:
```suggestion
    `params` contains key-value pairs that map values to the **params.keys** in a script.
    When you invoke a script with `params`, InfluxDB passes the values as  invocation parameters to the script.
```

Comment (jstirnaman) [reply src/svc/invocable-scripts/schemas/ScriptInvocationParams.yml:8]: Explanation is a work-in-progress.

**Review by jstirnaman — CHANGES_REQUESTED** [review]
Some suggestions, but mostly concerned about that `name` request param.

File: src/svc/invocable-scripts/schemas/Error.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+  properties:
```
Comment (jstirnaman) [inline src/svc/invocable-scripts/schemas/Error.yml:1]: We shouldn't need to copy Error and ErrorCode just for this svc.

File: src/svc/invocable-scripts/schemas/ErrorCode.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+description: code is the machine-readable error code.
```
Comment (jstirnaman) [inline src/svc/invocable-scripts/schemas/ErrorCode.yml:1]: We shouldn't need to copy Error and ErrorCode just for this svc.

**Review by jstirnaman — CHANGES_REQUESTED** [review]

File: src/svc/invocable-scripts/schemas/ScriptHTTPResponseData.yml [hunk 5]
```diff
@@ -1,3 +1,5 @@
-description: The data sent in the response body when a script is invoked by an HTTP request. User defined and dynamic.
+description: |
+  The response body contains the results of the executed script.
+  The response is user defined and dynamic.
```
Comment (jstirnaman) [inline src/svc/invocable-scripts/schemas/ScriptHTTPResponseData.yml:3]:
```suggestion
  The response is user-defined and dynamic.
```

**Review by jstirnaman — APPROVED** [review]
One minor fix.

**Pull request merged**